_data/simple-icons.json

```
@@ -6566,6 +6566,12 @@
             "source": "http://www.netapp.com/",
             "guidelines": "https://www.netapp.com/company/legal/trademark-guidelines/"
         },
+        {
+            "title": "NetBox",
+            "hex": "1685FC",
+            "source": "https://netbox.readthedocs.io/en/stable/development/style-guide/",
```

```suggestion
            "source": "https://netbox.readthedocs.io",
```

The logo SVG itself can be found on the homepage.